### ISSUE TITLE
Increase max listeners

### ISSUE THREAD

**Issue by v-stickykeys** [ceramicnetwork/js-ipfs-daemon#16]
We are getting this warning which we still want but at a higher level

```
(node:27) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [Socket]. Use emitter.setMaxListeners() to increase limit
```

**Comment by v-stickykeys** [comment]
Will try 50 but may be relative to number peers connected (limits are 200-500)

**Comment by v-stickykeys** [comment]
@zachferland haven't found a way to do this. do you know how?

**Comment by stbrody** [comment]
Maybe `process.setMaxListeners`?  https://stackoverflow.com/questions/9768444/possible-eventemitter-memory-leak-detected

**Comment by oed** [comment]
That would only work in nodejs? Maybe that's fine since this repo will only run on our infra?

**Comment by stbrody** [comment]
> That would only work in nodejs? Maybe that's fine since this repo will only run on our infra?

I don't really understand what you're getting at here.  Only work in nodejs as opposed to what?  This seems like a javascript-specific issue to begin with?  Or do you mean server-side nodejs as opposed to in-browser?

**Comment by oed** [comment]
@stbrody I assume the `process` variable is only available in nodejs, and not in browsers.

**Comment by stbrody** [comment]
I just found [this issue](https://github.com/ipfs/js-ipfs/issues/1597), which suggests that `ipfs.pubsub.setMaxListeners` might do the trick

**Comment by v-stickykeys** [comment]
> I just found [this issue](https://github.com/ipfs/js-ipfs/issues/1597), which suggests that `ipfs.pubsub.setMaxListeners` might do the trick

@stbrody yeah I tried this but it said `setMaxListeners` did not exist on it

**Comment by v-stickykeys** [comment]
> That would only work in nodejs? Maybe that's fine since this repo will only run on our infra?

yeah exactly @oed this would be the ipfs instances we run on our infra
